source/11-extending/05-reserved-values-available-for-extending-reports.rst

```diff
@@ -44,6 +44,11 @@ COUNTER recognizes that there are some common extensions that content providers
      - TR
      - PDF
 
+   * - Book_Segment_Count
+     - The number of Book_Segments within a Book. Note that this extension MUST only be used where the number of Book_Segments is known, and MUST NOT be used where the number of Book_Segments is only being estimated.
+     - PR, DR, TR
```

Comment:
PR and DR seem wrong to me, since the number of Book_Segments can only be specified on the title level.